rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -666,111 +682,119 @@ namespace rocal{
+        m.def("Brightness", &rocalBrightness,
             py::return_value_policy::reference);
-        m.def("CenterCropFixed",&rocalCropCenterFixed,
+        m.def("BrightnessFixed", &rocalBrightnessFixed,
             py::return_value_policy::reference);
-        m.def("Brightness",&rocalBrightness,
+        m.def("GammaCorrection", &rocalGamma,
             py::return_value_policy::reference);
-        m.def("GammaCorrection",&rocalGamma,
+        m.def("Rain", &rocalRain,
             py::return_value_policy::reference);
-        m.def("Rain",&rocalRain,
+        m.def("Snow", &rocalSnow,
             py::return_value_policy::reference);
-        m.def("Snow",&rocalSnow,
+        m.def("Blur", &rocalBlur,
             py::return_value_policy::reference);
-        m.def("Blur",&rocalBlur,
+        m.def("Contrast", &rocalContrast,
             py::return_value_policy::reference);
-        m.def("Contrast",&rocalContrast,
+        m.def("Flip", &rocalFlip,
             py::return_value_policy::reference);
-        m.def("Flip",&rocalFlip,
+        m.def("Jitter", &rocalJitter,
             py::return_value_policy::reference);
-        m.def("Jitter",&rocalJitter,
+        m.def("Rotate", &rocalRotate,
             py::return_value_policy::reference);
-        m.def("Rotate",&rocalRotate,
+        m.def("Hue", &rocalHue,
             py::return_value_policy::reference);
-        m.def("Hue",&rocalHue,
+        m.def("Saturation", &rocalSaturation,
             py::return_value_policy::reference);
-        m.def("Saturation",&rocalSaturation,
+        m.def("WarpAffineFixed", &rocalWarpAffineFixed,
             py::return_value_policy::reference);
-        m.def("WarpAffine",&rocalWarpAffine,
+        m.def("Fog", &rocalFog,
             py::return_value_policy::reference);
-        m.def("Fog",&rocalFog,
+        m.def("FishEye", &rocalFishEye,
             py::return_value_policy::reference);
-        m.def("FishEye",&rocalFishEye,
+        m.def("Vignette", &rocalVignette,
             py::return_value_policy::reference);
-        m.def("Vignette",&rocalVignette,
+        m.def("SnPNoise", &rocalSnPNoise,
             py::return_value_policy::reference);
-        m.def("SnPNoise",&rocalSnPNoise,
+        m.def("Exposure", &rocalExposure,
             py::return_value_policy::reference);
-        m.def("Exposure",&rocalExposure,
+        m.def("Pixelate", &rocalPixelate,
             py::return_value_policy::reference);
-        m.def("Pixelate",&rocalPixelate,
+        m.def("Blend", &rocalBlend,
             py::return_value_policy::reference);
-        m.def("Blend",&rocalBlend,
+        m.def("RandomCrop", &rocalRandomCrop,
             py::return_value_policy::reference);
-        m.def("RandomCrop",&rocalRandomCrop,
+        m.def("ColorTemp", &rocalColorTemp,
             py::return_value_policy::reference);
-        m.def("ColorTemp",&rocalColorTemp,
+        m.def("LensCorrection", &rocalLensCorrection,
```

Comment:
Why are the other functions removed?

Reply:
The functions are not removed Shobana. They were reordered so it looks like the calls were removed in the file changes